### PR TITLE
6 get /api/articles/article id/comments

### PR DESCRIPTION
**Pull request by plaintree**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -137,3 +139,68 @@ describe("5. GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("6. GET /api/articles/:article_id/comments", () => {
+  it("status:200, should responds with an array of comment objects", () => {
```
Comment (B7Bprog) [inline __tests__/app.test.js]: In the test description please include that comments are sorted by most recent.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -137,3 +139,68 @@ describe("5. GET /api/articles/:article_id", () => {
...
+  it("status:404, should responds with error message when the path is not found", () => {
+    return request(app)
+      .get("/api/articles/1/commentsssssss")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Route not found");
+      });
+  });
+
```
Comment (B7Bprog) [inline __tests__/app.test.js:184]: Please write another test of status 200 where the article ID exists but there are no comments, so need to expect an empty array.

**Review by B7Bprog — COMMENTED** [review]
Hey, you are doing well, but please read my comments and amend your code, then send us another pr. 👍

File: __tests__/app.test.js (hunk 2 above)
Comment (plaintree) [reply __tests__/app.test.js:184]: added underneath the first status 200 response

File: __tests__/app.test.js [hunk 3]
```diff
@@ -137,3 +137,78 @@ describe("5. GET /api/articles/:article_id", () => {
...
+  it("status:404, should responds with error message when the path is not found", () => {
+    return request(app)
+      .get("/api/articles/1/commentsssssss")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Route not found");
+      });
```
Comment (Poonam-raj) [inline __tests__/app.test.js:182]: Hey 👋🏾  you don't need to do anymore 'bad path' testing for future endpoints, you've got a few tests checking that your bad paths are handled and we are now confident that non-existent endpoints are handled.

File: __tests__/app.test.js [hunk 4]
```diff
@@ -137,3 +137,78 @@ describe("5. GET /api/articles/:article_id", () => {
...
+  });
+
+  it("status:400, should responds with error message when article_id is out of range of type integer", () => {
+    return request(app)
+      .get("/api/articles/1234523423432423/comments")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Out Of Range For Type Integer");
+      });
```
Comment (Poonam-raj) [inline __tests__/app.test.js:212]: Love the error testing!

**Review by Poonam-raj — APPROVED** [review]
Looks great! one comment on the bad path testing (you don't need to do it for future endpoints) but otherwise great. Feel free to merge
